SampleApp/Sources/UI/Views/DisplayViews/DisplayViewCell/DisplayListViewCell.swift

```diff
@@ -27,6 +27,13 @@ final class DisplayListViewCell: UITableViewCell {
     @IBOutlet private weak var titleLabel: UILabel!
     @IBOutlet private weak var subTitleLabel: UILabel!
     
+    @IBOutlet private weak var imageToggle: UIButton!
+    @IBOutlet private weak var textToggle: UIButton!
+    
+    private var token: String?
+    
+    var onToggleSelect: ((String) -> Void)?
```

Comment:
closure 파라미터에 이름을 붙여주는게 어떨까요-?
```suggestion
    var onToggleSelect: ((token: String) -> Void)?
```

Reply:
좋은 생각입니다. 반영했습니다